.github/workflows/cherry-pick-comment.yml

```diff
@@ -0,0 +1,25 @@
+name: Cherry Pick On Comment
+on:
+  issue_comment:
+    types: [created]
+jobs:
+  cherry-pick:
+    name: Cherry Pick
+    if: |
+      github.event.issue.pull_request != '' &&
+      contains(github.event.comment.body, '/cherry-pick') &&
+      ((github.event.pull_request.author_association != 'CONTRIBUTOR') ||
+            (github.event.pull_request.author_association != 'MEMBER') ||
+            (github.event.pull_request.author_association != 'OWNER') ||
+             (github.event.pull_request.author_association != 'COLLABORATOR'))
```

Comment:
does it mean that other user associations like `FIRST_TIMER` work? 
why isn't `github.event.pull_request.author_association == 'MEMBER'` enough?

i.e anyone in the org can add or `CONTRIBUTOR` because it on a pr that was merged by a contributing user?